app/components/main-text.tsx

```
@@ -1,25 +1,94 @@
 
 export default function MainText({}: Props) {
-  const [mainText, setMainText] = useState("write this as fast as possible");
-  const mainTextList = mainText.split("");
-  return (
-    <>
-      {mainTextList.map((character: string, index: number) => {
-        if (character !== " ") {
-          return (
-            <CharacterText key={index + character} character={character} />
-          );
+  const [mainText, setMainText] = useState<string | null>(
```

ESLint

[error] 'setMainText' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)